internal/pkg/vault/secrets_test.go

```diff
@@ -45,11 +45,11 @@ const (
 	// define as constants to avoid using global variables as global variables are evil to the whole package level scope:
 	// Global variables can cause side effects which are difficult to keep track of. A code in one function may
 	// change the variables state while another unrelated chunk of code may be affected by it.
-	testPath      = "/secret1"
-	testPath2     = "/secret2"
-	testPath3     = "/secret3"
-	testPath4     = ""
-	testNamespace = "database"
+	testSecretName  = "/secret1"
+	testSecretName2 = "/secret2"
+	testSecretName3 = "/secret3"
```

Comment:
We need the code to not require `/` at beginning of the name. The internal code should add it.
That should be done here: https://github.com/edgexfoundry/go-mod-secrets/blob/main/pkg/types/config.go#L50
and subPath should be changed everywhere.